src/Features/CSharp/Portable/CodeRefactorings/InlineTemporary/InlineTemporaryCodeRefactoringProvider.cs

```diff
@@ -69,7 +69,7 @@ internal partial class InlineTemporaryCodeRefactoringProvider : CodeRefactoringP
                 return;
             }
 
-            if (variableDeclarator.Initializer.RefKeyword.Kind() != SyntaxKind.None)
+            if (variableDeclarator.Initializer.Kind() == SyntaxKind.RefExpression)
             {
                 // TODO: inlining byref temps is NYI
```

Comment:
Yes, this line of code maintains the previous behavior.

---
In reply to: [73059077](https://github.com/dotnet/roslyn/pull/12841#discussion_r73059077) [](ancestors = 73059077)